src/compose.ts

```diff
@@ -493,7 +493,6 @@ export interface ComposeFactory<T, O extends Options> extends ComposeMixinable<T
 	/**
 	 * Extend the factory prototype with the supplied object literal, class, or factory
 	 *
-	 * @deprecated
```

Comment:
With `createMixin` , the result is not a factory at all. It can't be invoked and just records the arguments provided to it, essentially replaying them on a target factory later.